### DISCUSSION TITLE
Twitter problems

### DISCUSSION THREAD

**Discussion by Ckaarraa** [IBAX-io/go-ibax#1920] [Q&A]
Hi my twitter banned . Im not logging. What can i do? Please help me.

**Comment by yolandadadada** [comment]
can you send the Screenshot here？

**Reply by Ckaarraa** [reply]
![305A1681-BB07-4069-AFB6-8095BB3CCD4C](https://user-images.githubusercontent.com/97296238/155124263-8548b92e-31b3-486b-be7f-91f98227ae77.png)
![59D36971-E7FD-4402-B221-21BD1EEB5696](https://user-images.githubusercontent.com/97296238/155124273-b3a3ca7c-5578-4fad-9d7a-272e60cc57eb.png)


@airdrop2sniffer username
